plugins/android/src/main/java/dataBinding.kt

```diff
@@ -1,123 +0,0 @@
-import tools.forma.android.validation.disallowResources
-import tools.forma.android.visibility.Public
-import tools.forma.android.visibility.Visibility
-import org.gradle.api.Project
-import tools.forma.android.target.UiLibraryTargetTemplate
-import tools.forma.deps.core.applyDependencies
-import tools.forma.deps.core.FormaDependency
-import tools.forma.validation.validate
-
-/**
- * Android Data Binding Target - Data Binding layouts collection
- *
- * Only MVVM interfaces and layouts allowed, no business logic or other resource types
- *
- * @param packageName - Application package name, used for publishing
- * @param owner - owner of the target, team responsible for maintenance
- * @param visibility - visibility scope for the target
- * @param dependencies - list of external and project dependencies for the target
- * @param consumerMinificationFiles - Proguard/R8 minification files list
- */
-fun Project.dataBinding(
-    packageName: String,
-    owner: Owner = NoOwner,
-    visibility: Visibility = Public,
-    dependencies: FormaDependency = emptyDependency(),
-    consumerMinificationFiles: Set<String> = emptySet()
-) {
-    checkDataBindingFlag()
-
-    onlyAllowLayouts()
-
-    target.validate(DataBindingTargetTemplate)
-    val libraryFeatureConfiguration = AndroidLibraryFeatureConfiguration(
-        packageName = packageName,
-        consumerMinificationFiles = consumerMinificationFiles,
-        dataBinding = true
-    )
-    applyFeatures(
-        androidLibraryFeatureDefinition(libraryFeatureConfiguration),
-        kotlinAndroidFeatureDefinition(),
-        kotlinKaptFeatureDefinition()
-    )
-    applyDependencies(
-        validator = validator(
-            ApiTargetTemplate,
-            UiLibraryTargetTemplate,
-            WidgetTargetTemplate,
-            AndroidUtilTargetTemplate,
-            DataBindingAdapterTargetTemplate,
-            ResourcesTargetTemplate,
-            LibraryTargetTemplate
-        ),
-        dependencies = dependencies,
-        repositoriesConfiguration = Forma.settings.repositories
-    )
-}
-
-/**
- * Android Data Binding Adapters - Shared Data Binding Adapters collection
- *
- * Only DataBinding adapters allowed, no resources or business logic
- *
- * @param packageName - Application package name, used for publishing
- * @param owner - owner of the target, team responsible for maintenance
- * @param visibility - visibility scope for the target
- * @param dependencies - list of external and project dependencies for the target
- * @param consumerMinificationFiles - Proguard/R8 minification files list
- */
-fun Project.dataBindingAdapters(
-    packageName: String,
-    owner: Owner = NoOwner,
-    visibility: Visibility = Public,
-    dependencies: FormaDependency = emptyDependency(),
-    consumerMinificationFiles: Set<String> = emptySet()
-) {
-    checkDataBindingFlag()
-
-    disallowResources()
-
-    target.validate(DataBindingAdapterTargetTemplate)
```

Comment:
I think that `DataBindingAdapterTargetTemplate` and `DataBindingTargetTemplate` should also be deleted

Reply:
Done